src/test/kotlin/no/nav/familie/ef/sak/beregning/skolepenger/SkolepengerMaksbeløpTest.kt

```diff
@@ -50,17 +60,7 @@ internal class SkolepengerMaksbeløpTest {
     @Nested
     inner class MaksBeløpEtÅrFremITiden {
 
-        private val ÅR_OM_2_ÅR = Year.of(2024)
-
-        @Test
-        internal fun `maksbeløp for universitet et år frem i tiden skal returnere forrige år sitt verdi`() {
-            assertThat(maksbeløpForÅr(HØGSKOLE_UNIVERSITET, Year.of(2023))).isEqualTo(69_500)
-        }
-
-        @Test
-        internal fun `maksbeløp for videregående et år frem i tiden skal returnere forrige år sitt verdi`() {
-            assertThat(maksbeløpForÅr(VIDEREGÅENDE, Year.of(2023))).isEqualTo(29_002)
-        }
```

Comment:
Spør av nysgjerrighet :) Hvorfor fjernet du disse i stedet for å oppdatere årstallene?

Reply:
Blir så klønete etter nyttår - da er ikke 2024 neste år lenger... Litt som den ovenfor "ÅR_OM_2_ÅR" - det er ikke to år til 2024.